### PR TITLE
Add workload validation support to RequestAuthentication

### PR DESCRIPTION
**Pull request by xeviknal**
Add basic support to RequestAuthentication objects:

- Multiple ns-wide objects found
- Two objects pointing to the same workload
- Workload not found

Screenshot of Kiali Console (30)

Testing scenarios: https://github.com/xeviknal/istio-lab/blob/master/2543.yaml

Fixes https://github.com/kiali/kiali/issues/2543

### CODE REVIEW

**Comment by xeviknal** [comment]
I put the DNM label in this one. I'd prefer to aggregate this validation into a generic one and therefore kiali.io won't have 3 duplicated validation descriptions per each object using it.

**Comment by xeviknal** [comment]
Documentation PR has been created: https://github.com/kiali/kiali.io/pull/279. Removing the DNM.
In this PR, sidecars, peerauthentication and AuthorizationPolicies validations should be tested. Specially the ones regarding to the workload checks (same as description).

File: business/istio_validations.go [hunk 1]
```diff
@@ -463,6 +466,15 @@ func (in *IstioValidationsService) fetchDetails(rValue *kubernetes.IstioDetails,
 			}
 			go fetchIstioObjects(&istioDetails.Sidecars, namespace, getSidecars, &wg2, errChan2)
 		}
+		if nsCached && kialiCache.CheckIstioResource(kubernetes.RequestAuthentications) {
```
Comment (lucasponce) [inline business/istio_validations.go:469]: This is good.
No blocker, probably good to have a follow up PR to add all validations objects to the cache.
For Istio, only it's cached VirtualServices,DestinationRules,Gateways and Sidecars, perhaps we can add more.
(It's also a tradeoff).

I can prepare a next PR for that, too.

**Comment by prachiyadav** [comment]
Screenshot from 2020-08-07 16-48-15

**Pull request merged**